server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -285,6 +285,8 @@ public void run() {
               try {
                 alog.close();
               } catch (Exception e) {
+                // Throw an Error, not an Exception, so the AccumuloUncaughtExceptionHandler
+                // will log this then halt the VM.
```

Comment:
It looks like this comment was pasted into the wrong place. This is where the error is logged. The throw is further down, and is missing this comment.

But, I am wondering if InterruptedException is one of the checked exceptions possible from this overly broad catch clause. We really shouldn't have overly broad catch clauses this way. It's much better to do multi-catch to enumerate the exceptions we want to handle here. RTEs can be included, if the catch is intended to handle them as well.

For example:

```java
  } catch (IOException | InterruptedException | RuntimeException e) {
```

Reply:
Yes, copy/paste error. I will fix. I'll look at the close method and see if IntExc is possible.